CHANGELOG.md

```diff
@@ -17,8 +17,10 @@
 - Fix concurrent execution of CLI function `create_all_indicators` using async and semaphores ([#254])
 - Support choosing a single indicator and/or single layer for CLI command `create_all_indicators` ([#254])
 - Indicators based on GHS-POP use raster file stored on disk instead of raster in the database ([#276])
+- Reverse [#221], rename `ideal_vgi_lulc` layer and return result description in the case that the Mapping Saturation Indicator could not been calculated ([#226])
```

Comment:
Did you maybe mean _revert_ instead of _reverse_?

Reply:
I meant revert. Thanks.